release-notes.html

```diff
@@ -64,6 +64,9 @@ <h2>Fixes and Improvements</h2>
 
 
 <ul>
+    <li>
+        <a href="https://github.com/wso2/product-is/milestone/104?closed=1">5.10.0-M9</a>
```

Comment:
Is it 5.10.0-M9 or alpha?

Reply:
Fixed in 48676aa2253b1948b3e0f3c13b6c38f003b42f6f